packages/task/src/browser/task-definition-registry.spec.ts

```diff
@@ -54,6 +55,52 @@ describe('TaskDefinitionRegistry', () => {
             }
         }
     };
+    const FOOBAR_TASK_META = {
+        TYPE: 'foobar_type',
+        SRC: 'foobar_src'
+    };
+    const defaultPresentation = {
+        clear: false,
+        echo: true,
+        focus: false,
+        panel: PanelKind.Shared,
+        reveal: RevealKind.Always,
+        showReuseMessage: true,
+    };
+    const foobarTaskFixture = {
```

Comment:
minor: what does a `fixture` represent, it is the first time I heard of this terminology for `tasks`.

Reply:
Nice point. Well, frankly I didn't aim to use it exclusively for `tasks`, but I mean to use "test fixtures" here. In my understanding `fixture` can be something stable across runs of tasks. E.g., a file loaded with sample data or a script setting up stable env for task. 

Perhaps here I need to change it into mock or fake.